### DISCUSSION TITLE
Adding my own info/help page to the Qfield app home page

### DISCUSSION THREAD

**Discussion by mahmoodi-source** [opengisch/QField#5128] [Ideas]
I want to add my own information or help page to the Qfield app home page, so I can add my own information or other data links to help users. Is it possible to add our own information or help page to the Qfield App home page?

**Comment by suricactus** [comment]
Thanks for the question. No, it is not currently possible to add additional custom info on the app home page. If you want to discuss the possibility of getting this implemented, please send an email to sales [at] qfield.org .

**Reply by mahmoodi-source** [reply]
ok. thanks for your good informations.

**Comment by nirvn** [comment]
One thing QField isn't (yet? :) ) leveraging is the project metadata.  There's plenty of room there to add useful information in the abstract section. That'd be worth implementing.

**Comment by ZsanettMed** [comment]
Hello @mahmoodi-source ,

Thank you for your feedback. Feel free to share your idea(s) on the updated [QField Idea Platform](https://ideas.qfield.org/app-feature-requests). This lets other users vote on them and helps us prioritize the roadmap. 
We're transferring all ideas to the platform, so I'll be closing this ticket.

Thanks again and best,
Zsanett
